### ISSUE TITLE
--enable=mergePaths breaks file

### ISSUE THREAD

**Issue by JoKalliauer** [svg/svgo#1048]
Prozessing https://commons.wikimedia.org/wiki/File:Insignia_of_the_Indonesian_Army.svg

> svgo -i [input.svg.txt](https://github.com/svg/svgo/files/2435622/input.svg.txt) -o [output.svg.txt](https://github.com/svg/svgo/files/2435624/output.svg.txt) --pretty --indent=1

# Input
[input.svg.txt](https://github.com/svg/svgo/files/2435622/input.svg.txt)
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="499.25" height="732.44">
 <g fill="#ffe900" fill-rule="evenodd" stroke="#1b1918">
  <g stroke-width="2.52">
   <path d="M373.27 534.98c-8.092-54.74-4.391-98.636 56.127-90.287 77.894 55.595-9.147 98.206-5.311 151.74 21.027 45.08 17.096 66.495-7.512 68.302-17.258 10.998-32.537 13.238-46.236 8.48-.246-1.867-.69-3.845-1.368-5.94l-19.752-40.751c44.709 19.982 82.483-.171 51.564-24.28zm32.16-40.207c-5.449-9.977 3.342-14.397 8.048-3.55 12.4 31.857 6.043 40.206-16.136 72.254l-1.911-2.463c11.558-13.292 20.249-27.75 21.334-39.194.899-9.481-5.973-16.736-11.335-27.048z"/>
   <path d="M407.72 580.04c40.745 49.516-3.991 92.385-40.977 82.64"/>
   <path d="M423.3 626.43c4.321 5.031 6.597 22.07-6.727 38.302"/>
   <path d="M356.53 527.67l15.49-17.261c.791 11.047 10.586 19.14 6.804 26.351.851-3.058 7.049 3.802 3.565 8.906.431 1.443.947-.572 2.333.458 1.065.791 1.223 3.789 2.698 5.03 3.244 2.731 8.327 4.33 11.421 7.282 5.45 5.2 8.492 14.198 11.828 20.258 8.067 14.65 4.153 31.147-9.82 28.766-3.633 3.071-7.683 2.855-8.838-.206-5.601 8.117-11.785 7.961-18.367 2.146 2.716-1.547 7.498-2.947 8.886-7.886 1.388-4.94-3.124-16.666-.556-21.754-2.345-2.867-3.264-9.455-6.205-12.011-4.162-3.617-10.156-3.303-10.782-8.762.12 4.459-2.012 6.539-5.289 7.411a409.571 409.571 0 0 0-24.003-6.239c2.421-10.747 8.111-15.815 22.149-6.383-3.8-6.821 2.037-10.143 5.988-6.562-3.034-1.928-2.627-7.437 1.477-7.918-3.466.123-6.344-7.775-8.78-11.625z"/>
  </g>
  <g stroke-width="2.52">
   <path d="M127.69 535.38c8.095-54.74 4.392-98.636-56.149-90.287-77.924 55.595 9.15 98.206 5.313 151.74-21.035 45.08-17.103 66.495 7.515 68.302 16.44 10.473 31.085 13.004 44.28 9.114.066-2.685.446-5.67 1.2-8.978l18.658-36.964c-43.276 17.616-78.519-2.138-48.34-25.663zm-32.172-40.207c5.451-9.977-3.343-14.397-8.051-3.55-12.404 31.857-6.045 40.206 16.142 72.254l1.912-2.463c-11.563-13.292-20.257-27.75-21.342-39.194-.899-9.481 5.975-16.736 11.339-27.048z"/>
   <path d="M93.22 580.44c-39.424 47.892 1.15 89.561 37.331 83.426"/>
   <path d="M77.64 626.83c-4.323 5.031-6.599 22.07 6.73 38.302"/>
   <path d="M144.43 528.07l-15.496-17.261c-.791 11.047-10.59 19.14-6.807 26.351-.851-3.058-7.052 3.802-3.566 8.906-.431 1.443-.947-.572-2.334.458-1.065.791-1.224 3.789-2.699 5.03-3.245 2.731-8.329 4.33-11.425 7.282-5.452 5.2-8.495 14.198-11.832 20.258-8.07 14.65-4.155 31.147 9.824 28.766 3.634 3.071 7.686 2.855 8.842-.206 5.604 8.117 11.79 7.961 18.374 2.146-2.717-1.547-7.501-2.947-8.89-7.886-1.389-4.94 3.126-16.666.556-21.754 2.346-2.867 3.265-9.455 6.207-12.011 4.164-3.617 10.159-3.303 10.786-8.762-.117 4.361 1.922 6.444 5.078 7.349a415.436 415.436 0 0 1 24.19-6.333c-2.442-10.64-8.147-15.614-22.122-6.229 3.801-6.821-2.038-10.143-5.991-6.562 3.035-1.928 2.628-7.437-1.477-7.918 3.467.123 6.347-7.775 8.783-11.625z"/>
  </g>
  <path stroke-width="1.89" d="M105.18 568.05c-5.749 7.187-14.95 22.794-5.079 39.81m15.039-41.72c-5.75 7.187-14.007 28.339-6.202 41.519m63.872-90.569c-7.867-6.087-25.742 8.591-25.068 22.884 11.489 2.651 31.989-14.472 25.068-22.884z"/>
 </g>
</svg>
```
![input](https://user-images.githubusercontent.com/15687581/46315330-72376280-c5cd-11e8-9749-6bd754519208.png)

# Output
[output.svg.txt](https://github.com/svg/svgo/files/2435624/output.svg.txt)
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="499.25" height="732.44">
 <g fill="#ffe900" fill-rule="evenodd" stroke="#1b1918">
  <g stroke-width="2.52">
   <path d="M373.27 534.98c-8.092-54.74-4.391-98.636 56.127-90.287 77.894 55.595-9.147 98.206-5.311 151.74 21.027 45.08 17.096 66.495-7.512 68.302-17.258 10.998-32.537 13.238-46.236 8.48-.246-1.867-.69-3.845-1.368-5.94l-19.752-40.751c44.709 19.982 82.483-.171 51.564-24.28zm32.16-40.207c-5.449-9.977 3.342-14.397 8.048-3.55 12.4 31.857 6.043 40.206-16.136 72.254l-1.911-2.463c11.558-13.292 20.249-27.75 21.334-39.194.899-9.481-5.973-16.736-11.335-27.048zM407.72 580.04c40.745 49.516-3.991 92.385-40.977 82.64M423.3 626.43c4.321 5.031 6.597 22.07-6.727 38.302"/>
   <path d="M356.53 527.67l15.49-17.261c.791 11.047 10.586 19.14 6.804 26.351.851-3.058 7.049 3.802 3.565 8.906.431 1.443.947-.572 2.333.458 1.065.791 1.223 3.789 2.698 5.03 3.244 2.731 8.327 4.33 11.421 7.282 5.45 5.2 8.492 14.198 11.828 20.258 8.067 14.65 4.153 31.147-9.82 28.766-3.633 3.071-7.683 2.855-8.838-.206-5.601 8.117-11.785 7.961-18.367 2.146 2.716-1.547 7.498-2.947 8.886-7.886 1.388-4.94-3.124-16.666-.556-21.754-2.345-2.867-3.264-9.455-6.205-12.011-4.162-3.617-10.156-3.303-10.782-8.762.12 4.459-2.012 6.539-5.289 7.411a409.571 409.571 0 0 0-24.003-6.239c2.421-10.747 8.111-15.815 22.149-6.383-3.8-6.821 2.037-10.143 5.988-6.562-3.034-1.928-2.627-7.437 1.477-7.918-3.466.123-6.344-7.775-8.78-11.625z"/>
  </g>
  <g stroke-width="2.52">
   <path d="M127.69 535.38c8.095-54.74 4.392-98.636-56.149-90.287-77.924 55.595 9.15 98.206 5.313 151.74-21.035 45.08-17.103 66.495 7.515 68.302 16.44 10.473 31.085 13.004 44.28 9.114.066-2.685.446-5.67 1.2-8.978l18.658-36.964c-43.276 17.616-78.519-2.138-48.34-25.663zm-32.172-40.207c5.451-9.977-3.343-14.397-8.051-3.55-12.404 31.857-6.045 40.206 16.142 72.254l1.912-2.463c-11.563-13.292-20.257-27.75-21.342-39.194-.899-9.481 5.975-16.736 11.339-27.048zM93.22 580.44c-39.424 47.892 1.15 89.561 37.331 83.426M77.64 626.83c-4.323 5.031-6.599 22.07 6.73 38.302"/>
   <path d="M144.43 528.07l-15.496-17.261c-.791 11.047-10.59 19.14-6.807 26.351-.851-3.058-7.052 3.802-3.566 8.906-.431 1.443-.947-.572-2.334.458-1.065.791-1.224 3.789-2.699 5.03-3.245 2.731-8.329 4.33-11.425 7.282-5.452 5.2-8.495 14.198-11.832 20.258-8.07 14.65-4.155 31.147 9.824 28.766 3.634 3.071 7.686 2.855 8.842-.206 5.604 8.117 11.79 7.961 18.374 2.146-2.717-1.547-7.501-2.947-8.89-7.886-1.389-4.94 3.126-16.666.556-21.754 2.346-2.867 3.265-9.455 6.207-12.011 4.164-3.617 10.159-3.303 10.786-8.762-.117 4.361 1.922 6.444 5.078 7.349a415.436 415.436 0 0 1 24.19-6.333c-2.442-10.64-8.147-15.614-22.122-6.229 3.801-6.821-2.038-10.143-5.991-6.562 3.035-1.928 2.628-7.437-1.477-7.918 3.467.123 6.347-7.775 8.783-11.625z"/>
  </g>
  <path stroke-width="1.89" d="M105.18 568.05c-5.749 7.187-14.95 22.794-5.079 39.81m15.039-41.72c-5.75 7.187-14.007 28.339-6.202 41.519m63.872-90.569c-7.867-6.087-25.742 8.591-25.068 22.884 11.489 2.651 31.989-14.472 25.068-22.884z"/>
 </g>
</svg>
```
![output](https://user-images.githubusercontent.com/15687581/46315324-6cda1800-c5cd-11e8-8540-992284767cec.png)

----

<details>

<summary> Copyright </summary>

License: Public Domain

Author: https://id.wikipedia.org/wiki/Tentara_Nasional_Indonesia

Source: https://commons.wikimedia.org/wiki/File:Insignia_of_the_Indonesian_Army.svg

</details>

**Comment by JoKalliauer** [comment]
same problem for https://commons.wikimedia.org/wiki/File:Coat_of_arms_of_Paul_Henri_Delatte.svg (version of 09:13, 4. Sep. 2018)


#Input 
[Input.svg.txt](https://github.com/svg/svgo/files/2592079/Input.svg.txt)
![input](https://user-images.githubusercontent.com/15687581/48663967-05522c00-ea98-11e8-93f5-557fe290da51.png)


#Output
[Output.svg.txt](https://github.com/svg/svgo/files/2592080/Output.svg.txt)
![output](https://user-images.githubusercontent.com/15687581/48663969-0b480d00-ea98-11e8-9c9d-a452c6301ffd.png)

<details>

<summary>Copyright CC-BY-SA 4.0</summary>

Lizense: CC-BY-SA 4.0

Author: Olivier LPB and SajoR

Source: https://commons.wikimedia.org/wiki/File:Coat_of_arms_of_Paul_Henri_Delatte.svg

</details>

**Comment by TrySound** [comment]
Will close after release

**Comment by TrySound** [comment]
Fixed in 2.0.1
